config/action.d/mail-whois-lines.conf

```diff
@@ -4,6 +4,10 @@
 # Modified-By: Yaroslav Halchenko to include grepping on IP over log files
 #
 
+[INCLUDES]
+
+before = mail-whois-common.conf
```

Comment:
what about naming this one "mail-whois-autodetect.conf" and then having "mail-whois-basic.conf" which would simply define ```_whois_command = whois``` so if ran on systems e.g. without iconv and who knows what else (didn't try runing this one on OSX or some other systems yet), then could switch?  and add a comment there above about that?  I am ok to have -autodetect by default ;)